server/src/main/scala/drt/server/feeds/common/ArrivalFeed.scala

```diff
@@ -0,0 +1,30 @@
+package drt.server.feeds.common
+
+import akka.actor.ActorRef
+import akka.pattern.ask
+import akka.util.Timeout
+import drt.shared.FlightsApi.Flights
+import org.slf4j.{Logger, LoggerFactory}
+import server.feeds.{ArrivalsFeedFailure, ArrivalsFeedResponse, ArrivalsFeedSuccess, GetFeedImportArrivals}
+import services.SDate
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+case class ArrivalFeed(arrivalsActor: ActorRef)(implicit timeout: Timeout) {
+  val log: Logger = LoggerFactory.getLogger(getClass)
+
+  def requestFeed(portCode: String): Future[ArrivalsFeedResponse] =
+    arrivalsActor.ask(GetFeedImportArrivals)
+      .map {
+        case Some(Flights(arrivals)) =>
+          log.info(s"Got ${arrivals.length} $portCode port arrivals")
```

Comment:
I see the `portCode` is a relic of the LHRForecastFeed, but I think we can drop it from the log message since these logs are specific to the port anyway

Reply:
Initial I thought code is use for LHR and LGW but if we have separate instance running then we can remove it .